Blis/src/BlisBranchObjectInt.h

```diff
@@ -79,6 +79,15 @@ class BlisBranchObjectInt : public BcpsBranchObject {
             down_[1] = floor(value_);
             up_[0] = ceil(value_);
             up_[1] = model->getColUpper()[iColumn];
+	    //start:sahar
```

Comment:
This change does not affect BLIS in solving MIPs because we do not branch on integer-valued variables.